apps/etl-func/src/domain/message-status.ts

```diff
@@ -17,12 +17,12 @@ export const messageStatusIdSchema = z.string().min(1);
 
 export const messageStatusSchema = z
   .object({
-    fiscalCode: fiscalCodeSchema,
+    fiscalCode: fiscalCodeSchema.optional(),
     id: messageStatusIdSchema,
```

Comment:
is this messageStatusIdSchema really necessary? It is just a string

Reply:
It is exported because we use it also in messageStatusEventSchema, this way if we modify it here we don't have to modify it in the event

Reply:
Why don't we create the correct schema then? Something like this 
`const versionedId = z.string().regex(
  /^[0-9A-HJKMNP-TV-Z]{26}-\d+$/,
  'Invalid format. Expected <ULID-Number>'
);`